src/main/java/ch/heigvd/pro/b04/answers/AnswerController.java

```diff
@@ -360,7 +360,7 @@ public ServerAnswer updateAnswer(
     toUpdate.setTitle(answer.getTitle());
     toUpdate.setDescription(answer.getDescription());
 
-    return repository.save(toUpdate);
+    return toUpdate;
```

Comment:
Have we figured out when it's needed and not needed to push updates to the repository manually ?